corehq/apps/users/templates/users/roles_and_permissions.html

```diff
@@ -239,6 +244,14 @@ <h6 data-bind="visible: !name()">({% trans "No Name" %})</h6>
             </div>
           </td>
           {% endif %}
+          {% if request|toggle_enabled:"ATTENDANCE_TRACKING" %}
```

Comment:
Ditto
```suggestion
          {% if request|request_has_privilege:"ATTENDANCE_TRACKING" and request|toggle_enabled:"ATTENDANCE_TRACKING" %}
```

Reply:
Unrelated, but after a google search on the word "Ditto" I realised I've been using it all wrong my whole life!